### DISCUSSION TITLE
password not setting up

### DISCUSSION THREAD

**Discussion by shreyash729** [spencerwooo/onedrive-vercel-index#322] [Q&A] [answered]
![Capture1](https://user-images.githubusercontent.com/98596657/151594349-742b4bc8-2e93-4e99-834b-6934c02df68d.PNG)
![Capture](https://user-images.githubusercontent.com/98596657/151594366-552d5f97-fd13-477e-9bfc-6ebd5b49b761.PNG)

please help me i am not able to set password

**Comment by myl7** [comment]
To add a password-protected folder, you need to ont only add `.password` file, but also edit the `config/site.json` file to update field `protectedRoutes`. More is available in [the documentation](https://docs.spencerwoo.com/password-protected-folders).

**Comment by shreyash729** [comment]
![Capture](https://user-images.githubusercontent.com/98596657/151598489-215faba4-59bf-4e7e-ab01-a2a748f5d9e2.PNG)
every thing is fine i have checked it before but could not figure out the issue

**Comment by myl7** [comment, marked as answer]
Oh my miss.
Well, then try rename `.password.txt` to `.password`.

**Reply by shreyash729** [reply]
.txt is not in name it's the formate of file so how can I remove that

**Reply by myl7** [reply]
If you can access the OneDrive drive on Windows, you may enable "display filename with extension" on Windows and rename it to any name just like renaming a regular file.
The detailed tutorial is easy to find on Google.
OneDrive web page, surprisingly and sadly, lacks the functionality that allows you to rename the whole filename.
